modules/tinymce/src/core/test/ts/browser/ReadOnlyModeTest.ts

```diff
@@ -76,6 +81,12 @@ UnitTest.asynctest('browser.tinymce.core.ReadOnlyModeTest', (success, failure) =
       })
     ]);
 
+    const sAssertHrefOpt = (selector: string, expectedHref) => Step.sync(() => {
```

Comment:
Can we add a type here on `expectedHref`? It should be `Optional<string>` I believe.